src/pages/strategy/tabs/pvp/pvp.js

```diff
@@ -331,6 +339,13 @@
 						mapnum: 0,
 					};
 					
+					if(e.shiftKey) {
+						window.open("https://kc3kai.github.io/kancolle-replay/battleplayer.html#" + encodeURIComponent(JSON.stringify(encodeData), "_blank"));
```

Comment:
Looks cool. The only thing i want to mention though. Can it update buttons before opening new page?
Because otherwise it stays in "to site" mode. Not sure if it was part of the plan.